vip-jetpack/connection-pilot/class-jetpack-connection-pilot.php

```diff
@@ -86,6 +94,10 @@ public function schedule_cron() {
 		}
 	}
 
+	public function schedule_immediate_cron() {
+		wp_schedule_event( strtotime( '+1 minutes' ), self::CRON_SCHEDULE, self::CRON_ACTION );
```

Comment:
This will leave a perpetual job, instead can schedule single event: https://developer.wordpress.org/reference/functions/wp_schedule_single_event/